Makefile

```diff
@@ -86,9 +86,14 @@ verify-vendor:
 update-smith:
 	dep ensure -v -update github.com/atlassian/smith
 
+.PHONY: update-ctrl
+update-ctrl:
+	dep ensure -v -update github.com/atlassian/ctrl
+
 .PHONY: bump-dependencies
 bump-dependencies: \
```

Comment:
` dep ensure -v -update github.com/atlassian/ctrl  github.com/atlassian/smith` this should work faster.

Reply:
Done